package.json

```diff
@@ -2,37 +2,43 @@
   "name": "wikipedia-preview",
   "version": "1.8.0",
   "description": "Shows Wikipedia article preview in a popup",
-  "main": "dist/wikipedia-preview.production.js",
+  "main": "dist/wikipedia-preview.umd.cjs",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "module": "dist/wikipedia-preview.js",
+  "exports": {
+    ".": {
+      "import": "./dist/wikipedia-preview.js",
+      "require": "./dist/wikipedia-preview.umd.cjs"
+    }
+  },
   "scripts": {
     "test": "mocha --require ignore-styles,@babel/register,jsdom-global/register",
     "test:watch": "mocha --require ignore-styles,@babel/register,jsdom-global/register -w",
     "test:fix-lint": "eslint test --fix",
     "coverage": "nyc npm test",
-    "build:dev": "webpack --mode development",
-    "build:prod": "webpack --mode production",
-    "build": "npm run build:dev && npm run build:prod",
-    "dev": "webpack-dev-server --mode development --host=0.0.0.0 --disable-host-check",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "build": "vite build",
+    "dev": "vite",
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
     "wba": "npm run build:prod && npx webpack-bundle-analyzer dist/stats.json",
```

Comment:
Anyone know where is this `wba` command use? Trying to understand if it's safe to remove. Looks like it was added by Pita a couple of year ago

Reply:
yes i remember this, it's useful to understand how our lib size
> Visualize size of webpack output files with an interactive zoomable treemap.

I found the version in vite, you can now running `npm run vba` to see the result
https://github.com/wikimedia/wikipedia-preview/pull/177/commits/161fcf6592f956812aa49ddbd5506c823f9082f8